html/changelogs/FlashTweak.yml

```diff
@@ -0,0 +1,8 @@
+author: Scheveningen
+delete-after: True
+
+changes:
+  - balance: "Blinding sources are significantly increased in duration."
+  - rscdel: "Direct flashes no longer stun those who are -not- sentient trees, robots, or bugpeople."
+  - tweak: "Changed how flash break mechanics work. Crappy budget NT tech is more likely to break from overuse."
+  - tweak: "The security cyborg now sustains for longer with consistent use of the stun baton."
```

Comment:
This is a spaghetti changelog entry.

> -  tweak: "The security cyborg stun baton now consumes less energy."

Still should be moved to another PR.

Reply:
okiedoke.